src/ngsw-config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "index": "/index.html",
+  "assetGroups": [{
+    "name": "app",
+    "installMode": "prefetch",
+    "resources": {
+      "files": [
+        "/favicon.ico",
+        "/index.html"
+      ],
+      "versionedFiles": [
+        "/*.bundle.css",
+        "/*.bundle.js",
+        "/*.chunk.js"
+      ]
+    }
+  }, {
+    "name": "assets",
+    "installMode": "lazy",
+    "updateMode": "prefetch",
+    "resources": {
+      "files": [
```

Comment:
@Shekharrajak I'm glad you asked this. There is a `ngsw.json` which manages app's versions. Whenever a developer changes files locally, this file's version changes. Whenever, a user on the client side reloads his page, first only the request for `ngsw.json` would be made. If only it's found same, no other requests for files shall be made, else requests for new (or updated) files are made.  Hence, an improved loading speed.

Yes, this would work by serving the production build using `http-server`.

Reply:
> Yes, this would work by serving the production build using http-server.

So how did you tested it? I am curious to see the loading speed difference and check this out in http://pr-101-evalai.surge.sh/ 

Can you test it by changing some content and serve it again ? Offline it would be showing old content and after running the server again and after refreshing the page it should be able to update the changed content.